birdhouse/components/cowbird/default.env

```diff
@@ -68,5 +68,5 @@ DELAYED_EVAL="
 # (include this for Cowbird<2.0.0)
 #COMPONENT_DEPENDENCIES="
 #  $COMPONENT_DEPENDENCIES
-#  ./config/mongodb
+#  ./components/mongodb
 #"
```

Comment:
Technically, `component/magpie` should be in there, otherwise Cowbird doesn't really have anything to interact with.

Reply:
Should we even keep this since "this dependency is only required if the mongo instance is the one provided in config/mongodb (include this for Cowbird<2.0.0)"

Is cowbird still <2.0.0?  Else this comment block not needed anymore?

Reply:
I wouldn't worry about `cowbird<2.0.0`. No reason to go back. Only add the relevant current dependencies.

Reply:
That's what I thought, no need to go back to older cowbird so no need to keep this comment block about adding mongodb as dependencies.